CardScan/Classes/Torch.swift

```diff
@@ -19,22 +24,12 @@ struct Torch {
         } else {
             self.device = nil
         }
+        self.level = 1.0
     }
     
     mutating func toggle() {
         self.state = self.state == .on ? .off : .on
-
-        do {
-            try self.device?.lockForConfiguration()
-            if self.state == .on {
-                do { try self.device?.setTorchModeOn(level: 0.5) } catch { print("could not set torch mode on") }
-            } else {
-                self.device?.torchMode = .off
-            }
-            self.device?.unlockForConfiguration()
-        } catch {
-            print("error setting torch level")
-        }
+        self.setTorchMode()
     }
     
     mutating func luma(_ value: Double) {
```

Comment:
does this function get used anywhere? If so, where?

Reply:
i looked and it doesnt. you want me to get rid of it?

Reply:
you're talking about `luma` right?

Reply:
yeah luma. Let's kill it.